libraries/sdk/src/main/starfederation/datastar/clojure/adapter/common.clj

```diff
@@ -13,14 +13,14 @@
   - [[content-encoding]]
 
   With this 3 keys we can control buffering aspects of our SSE connction and
```

Comment:
"With this" -> "With these"

Reply:
Turns out there is another type in this line... Both are fixed.